Release v1.2.0

jasonbahl opened the pull request:
## TLDR

- updates the plugin to have conditional logic for the graphql_single_name and graphql_plural_name to only show and be required if show_in_graphql is set to true.
- cleans up code formatting
- DRY up the form markup that's used for the forms
- Fixes issue with postbox header markup

## More info

This is basically a complete refactor of the plugin. 

The overall goal is to handle the fields for GraphQL settings better. 

Currently, users can leave graphql_single_name and graphql_plural_name blank, but set the Post Type or Taxonomy to show_in_graphql and that causes issues. 

This makes the graphql_single_name and graphql_plural_name fields conditional to only show and be required if show_in_graphql is set to true. 

See example: 

![show-in-graphql-cptui](https://user-images.githubusercontent.com/1260765/112345168-6b63a500-8c8a-11eb-844a-45ef3c67a669.gif)

jasonbahl committed:
- updates the plugin to have conditional logic for the graphql_single_name and graphql_plural_name to only show and be required if show_in_graphql is set to true.
- cleans up code formatting
- DRY up the form markup that's used for the forms
- Fixes issue with postbox header markup

markkelnar commented on wp-graphql-custom-post-type-ui.php:
```diff
@@ -1,234 +1,357 @@
+			$this->show_in_graphql = false;
+		}
+
+		$this->show_in_graphql     = ! empty( $data['cpt_custom_post_type']['show_in_graphql'] ) ? $data['cpt_custom_post_type']['show_in_graphql'] : false;
```
These !isset() and empty() tests are confusing.  And won't $this->show_in_graphql conditions above become irrelevant because of this $this->show_in_graphql = .... on line 122?

jasonbahl replied:
reading this back through, I feel like I must have had other intentions that I didn't come back to 🤔 

Let me step back through.

jasonbahl replied:
@markkelnar I pushed up another commit fixing this. And ya, I was definitely in the middle of something and went a different direction and didn't come back to this. 

Thanks for the detailed review. Love it! 🙌

jasonbahl committed:
- Update Docblocks
- Remove unnecessary isset checks, and just clarify the ternary checks for the update callbacks
